golem_messages/message/__init__.py

```diff
@@ -54,6 +54,11 @@
 from golem_messages.message.concents import RejectReportComputedTask  # noqa
 from golem_messages.message.concents import VerdictReportComputedTask  # noqa
 from golem_messages.message.concents import FileTransferToken  # noqa
+from golem_messages.message.concents import ForceGetTaskResult  # noqa
+from golem_messages.message.concents import ForceGetTaskResultAck  # noqa
+from golem_messages.message.concents import ForceGetTaskResultFailed  # noqa
+from golem_messages.message.concents import ForceGetTaskResultRejected  # noqa
+from golem_messages.message.concents import ForceGetTaskResultUpload  # noqa
```

Comment:
Please don't import these messages here. Previous messages are imported for backwards compatibility only.